radixconfig.yaml

```diff
@@ -31,6 +31,8 @@ spec:
         CLUSTER_INGRESS_IPS: undefined # Comma separated list of IPs of all Public IP Prefixes assigned to the ClusterType (production, playground, development)
       secrets:
         - DYNATRACE_API_TOKEN # Token to use for display of availability metrics
+        - OAUTH2_CLIENT_ID # Azure Application ID
+        - OAUTH2_AUTHORITY # Azure Application authority
```

Comment:
remove from secrets, and move to variables
OAUTH2_AUTHORITY: https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0